starknet_py/tests/e2e/core/fixtures.py

```diff
@@ -0,0 +1,162 @@
+# pylint: disable=redefined-outer-name
+import pytest
+import pytest_asyncio
+
+from starknet_py.common import create_compiled_contract
+from starknet_py.constants import FEE_CONTRACT_ADDRESS
+from starknet_py.contract import Contract
+from starknet_py.hash.address import compute_address
+from starknet_py.net.account.account import Account
+from starknet_py.net.account.account_deployment_result import AccountDeploymentResult
+from starknet_py.net.client_models import DeclareTransactionResponse
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.models import StarknetChainId
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.net.udc_deployer.deployer import Deployer
+from starknet_py.tests.e2e.fixtures.constants import MAX_FEE
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+from starknet_py.tests.e2e.utils import _get_random_private_key_unsafe
+
+
+@pytest.fixture(scope="package")
+def core_gateway_client() -> GatewayClient:
+    return GatewayClient(net="https://external.integration.starknet.io")
```

Comment:
It would be good to have it configurable, as for instance, right now testnet is preferred